frontend/micro-ui/web/webpack.config.js

```diff
@@ -23,10 +27,17 @@ module.exports = {
   optimization: {
     splitChunks: {
       chunks: "all",
+      minSize: 20000,
+      maxSize: 50000,
+      enforceSizeThreshold: 50000,
+      minChunks: 1,
+      maxAsyncRequests: 30,
```

Comment:
@anilsingha-eGov is this okay setting maxAsyncRequests to 30?

Reply:
yes @rahuldevgarg